api-1.11/src/main/java/org/openmrs/module/metadatadeploy/bundle/AbstractMetadataBundle1_11.java

```diff
@@ -0,0 +1,12 @@
+package org.openmrs.module.metadatadeploy.bundle;
+
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.module.metadatadeploy.descriptor.PatientIdentifierTypeDescriptor1_11;
+
+@OpenmrsProfile(openmrsVersion = "1.11.*")
```

Comment:
Did you mean to use `1.11.0`?

Reply:
Sure, being honest, I do not know when to use 1.11.0 and when to use 1.11.*. I just searched through openmrs code and I saw in other places was used 1.11.*

Reply:
Is this of help? https://github.com/openmrs/openmrs-core/blob/2.3.1/api/src/main/java/org/openmrs/module/ModuleUtil.java#L256-L261

Reply:
That makes a lot of sense! Then, for sure I meant 1.11.0. Thanks @dkayiwa !